src/ConnectedMode.UnitTests/Persistence/LegacySolutionBindingPathProviderTests.cs

```diff
@@ -19,67 +19,58 @@
  */
 
 using System;
-using System.IO;
-using Microsoft.VisualStudio.Shell.Interop;
 using SonarLint.VisualStudio.ConnectedMode.Persistence;
+using SonarLint.VisualStudio.Core;
 
 namespace SonarLint.VisualStudio.ConnectedMode.UnitTests.Persistence
 {
     [TestClass]
     public class LegacySolutionBindingPathProviderTests
     {
-        private Mock<IVsSolution> solution;
```

Comment:
Miscellaneous: changed the existing tests to remove the `TestInitialize` in favour of a `CreateTestSubject` method.